framework/Objects/Taxonomy.php

```diff
@@ -75,6 +75,14 @@
 	 */
 	protected $rewrite_singular = false;
 
+	/**
+	 * 'hierarchical' - true or false allow hierarchical urls
+	 * (implemented in Version 3.1) - defaults to false
+	 *
+	 * @var bool
+	 */
+	protected $is_hierarchical_url = false;
```

Comment:
please update comment - I don't think it's correct.

please rename property to rewrite_hierarchical